examples/imitation_learning_openbot/generate_episodes.py

```diff
@@ -42,71 +43,73 @@
     else:
         scene_ids = [args.scene_id]
 
+    # create dataframe
+    df_columns = ["scene_id", "start_location_x", "start_location_y", "start_location_z", "goal_location_x", "goal_location_y", "goal_location_z"]
+    df = pd.DataFrame(columns=df_columns)
+
     # iterate over all scenes
     for i, scene_id in enumerate(scene_ids):
 
         spear.log("Processing scene: " + scene_id)
 
         # change config based on current scene
         config.defrost()
+        config.SIMULATION_CONTROLLER.SCENE_ID = scene_id
+        config.freeze()
 
-        if scene_id == "kujiale_0000":
-            config.SIMULATION_CONTROLLER.SCENE_ID = scene_id
-            config.SIMULATION_CONTROLLER.MAP_ID   = scene_id + "_bake"
-
-            # kujiale_0000 has scene-specific config values
-            scene_config_file = os.path.realpath(os.path.join(os.path.dirname(__file__), "scene_config.kujiale_0000.yaml"))
-
-        elif scene_id == "warehouse_0000":
-            config.SIMULATION_CONTROLLER.SCENE_ID = scene_id
-            config.SIMULATION_CONTROLLER.MAP_ID   = scene_id
+        # create Env object
+        env = spear.Env(config)
 
-            # warehouse_0000 has scene-specific config values
-            scene_config_file = os.path.realpath(os.path.join(os.path.dirname(__file__), "scene_config.warehouse_0000.yaml"))
+        # reset the simulation
+        env_reset_info = {}
+        _ = env.reset(reset_info=env_reset_info)
+        assert "success" in env_reset_info
+            
+        # generate candidate points based out of args.num_episodes_per_scene
+        candidate_points = env.get_random_points(args.num_episodes_per_scene * args.num_candidate_points_per_episode)
 
-        else:
-            assert False
+        # obtain a reachable goal point for every candidate point
+        reachable_points = env.get_reachable_points(candidate_points.tolist())
 
-        config.merge_from_file(scene_config_file)
-        config.freeze()
+        # get trajectories for every candidate_points and corresponding reachable_points
```

Comment:
- `paths = get_paths(...)`

Reply:
Updated this. Also, corrected a typo in the indexing `candidate_best_indices  = np.argsort(candidate_scores)[:args.num_episodes_per_scene:-1]`. Instead used `candidate_best_indices  = np.argsort(candidate_scores)[::-1][:args.num_episodes_per_scene]`